src/Sylius/Bundle/CoreBundle/Installer/Executor/CommandExecutor.php

```diff
@@ -56,6 +57,30 @@ public function runCommand(string $command, array $parameters = [], ?OutputInter
         return $this;
     }
 
+    /**
+     * @param string[] $commands
+     */
+    public function runCommands(array $commands, OutputInterface $output, bool $displayProgress = true): void
```

Comment:
I'm wondering if you still use that https://github.com/Sylius/Sylius/pull/16785/files#diff-8050a36004460f434ed5d8a022d166c8eec804aa4def6b1fe4c4868cc1796deeR63
[12 h 11](https://sylius.slack.com/archives/D06883W2NKC/p1724839894497659?thread_ts=1724839751.467829&cid=D06883W2NKC)
cause you keep the AbstractInstallCommand and there is that method there.